<?xml version="1.0"?>
<launch>
  
  <include file="$(find husky_gazebo)/launch/husky_playpen.launch"/>
  <include file="$(find husky_viz)/launch/view_robot.launch"/>
  <include file="$(find husky_navigation)/launch/move_base_mapless_demo.launch"/>
  
  <node name="nav_husky_robot_node" pkg="nav_husky_robot" type="nav_husky_robot_node" output="screen"/>

</launch>
